params/ekf.yaml

```diff
@@ -0,0 +1,251 @@
+### ekf config file ###
+se_node:
+    ros__parameters:
```

Comment:
Pardon my ignorance, but the double underscore here is intentional/necessary?

Reply:
It is necessary as per yaml syntax. 
https://github.com/ros2/rcl/blob/master/rcl_yaml_param_parser/README.md